### PR TITLE
feat(mapping/theme): basic and control statuses add(w/o button) / transparent colors add

### PR DESCRIPTION
**Pull request by 32penkin**

### CODE REVIEW

File: packages/eva/mapping.json [hunk 1]
```diff
@@ -388,16 +388,16 @@
...
             "textColor": "text-control-color",
             "iconTintColor": "text-control-color",
             "iconMarginHorizontal": 4,
             "state": {
               "disabled": {
-                "borderColor": "border-basic-color-3",
-                "backgroundColor": "background-basic-color-3",
-                "textColor": "text-disabled-color",
-                "iconTintColor": "text-disabled-color"
+                "borderColor": "transparent",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Wouldn't this change make the button visually smaller? I guess it should not be transparent, but somethig closer to `color-basic-600-transparent-24` that is set to `backgroundColor` (if we still have issues in React Native with it)

File: packages/eva/mapping.json [hunk 2]
```diff
@@ -453,7 +453,7 @@
                   }
                 }
               },
-              "white": {
+              "control": {
                 "backgroundColor": "color-basic-100",
                 "borderColor": "color-basic-100",
                 "textColor": "color-primary-default",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Please refactor this to `text-primary-color`

File: packages/eva/mapping.json [hunk 3]
```diff
@@ -545,74 +545,68 @@
         "outline": {
           "mapping": {
             "state": {
+              "active": {
+                "backgroundColor": "color-primary-500-transparent-16"
+              },
               "disabled": {
-                "backgroundColor": "background-basic-color-2"
+                "backgroundColor": "color-basic-600-transparent-16",
+                "borderColor": "color-basic-600-transparent-24"
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Here I have the opposite question to the one above. Will it affect React Native?

File: packages/eva/mapping.json [hunk 4]
```diff
@@ -632,7 +626,8 @@
           "mapping": {
             "state": {
               "disabled": {
-                "backgroundColor": "transparent"
+                "backgroundColor": "color-basic-600-transparent-16",
+                "borderColor": "transparent"
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Same issue as for disabled Button

File: packages/eva/mapping.json [hunk 5]
```diff
@@ -646,7 +641,7 @@
                 "state": {
                   "active": {
                     "borderColor": "transparent",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Visual control size issue

File: packages/eva/mapping.json [hunk 6]
```diff
@@ -663,7 +658,7 @@
...
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Visual control size issue

File: packages/eva/mapping.json [hunk 7]
```diff
@@ -680,7 +675,7 @@
...
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Visual control size issue

File: packages/eva/mapping.json [hunk 8]
```diff
@@ -697,7 +692,7 @@
...
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Visual control size issue

File: packages/eva/mapping.json [hunk 9]
```diff
@@ -714,7 +709,7 @@
...
```
Comment (artyorsh) [inline packages/eva/mapping.json]: Visual control size issue

File: packages/eva/mapping.json [hunk 10]
```diff
@@ -1013,109 +1012,159 @@
...
               "indeterminate": {
                 "iconTintColor": "text-control-color",
                 "iconHeight": 3,
                 "iconBorderRadius": 1.5
               },
               "disabled": {
-                "borderColor": "border-basic-color-3",
-                "backgroundColor": "background-basic-color-2",
-                "textColor": "text-disabled-color"
+                "borderColor": "color-basic-600-transparent-24",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: React Native affect issue?

File: packages/eva/mapping.json [hunk 11]
```diff
@@ -1013,109 +1012,159 @@
...
+                "backgroundColor": "color-basic-600-transparent-8",
+                "textColor": "color-basic-600-transparent-48"
+              },
+              "checked.disabled": {
+                "borderColor": "color-basic-600-transparent-48",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: React Native affect issue?

File: packages/eva/mapping.json [hunk 12]
```diff
@@ -2371,102 +2424,155 @@
...
-                "borderColor": "color-primary-default",
-                "iconTintColor": "color-primary-default"
+                "backgroundColor": "transparent"
               },
               "disabled": {
-                "backgroundColor": "background-basic-color-2",
-                "borderColor": "border-basic-color-3",
-                "textColor": "text-disabled-color"
+                "backgroundColor": "color-basic-600-transparent-16",
+                "borderColor": "color-basic-600-transparent-8",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: React Native affect issue?

File: packages/eva/mapping.json [hunk 13]
```diff
@@ -2371,102 +2424,155 @@
...
               "checked.active": {
-                "borderColor": "color-primary-focus"
+                "backgroundColor": "background-basic-color-1"
               },
               "checked.disabled": {
-                "iconTintColor": "background-basic-color-4",
-                "textColor": "text-disabled-color"
+                "backgroundColor": "transparent",
+                "iconTintColor": "color-basic-600-transparent-48",
+                "borderColor": "color-basic-600-transparent-48",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: React Native affect issue?

File: packages/eva/mapping.json [hunk 14]
```diff
@@ -2371,102 +2424,155 @@
...
+                    "borderColor": "color-basic-100",
+                    "backgroundColor": "color-basic-800"
+                  },
+                  "checked.active": {
+                    "outlineBackgroundColor": "color-basic-100-transparent-16",
+                    "borderColor": "color-basic-100"
+                  },
+                  "disabled": {
+                    "backgroundColor": "color-basic-600-transparent-16",
+                    "borderColor": "color-basic-600-transparent-32"
```
Comment (artyorsh) [inline packages/eva/mapping.json]: React Native affect issue?

File: packages/eva/mapping.json [hunk 15]
```diff
@@ -3051,83 +3163,154 @@
...
             "state": {
-              "active": {
-                "borderColor": "border-basic-color-4",
-                "outlineBackgroundColor": "outline-color"
-              },
               "disabled": {
-                "borderColor": "border-basic-color-4",
-                "backgroundColor": "background-basic-color-3"
+                "thumbBackgroundColor": "color-basic-600-transparent-48",
+                "borderColor": "color-basic-600-transparent-32",
```
Comment (artyorsh) [inline packages/eva/mapping.json]: React Native affect issue?

File: packages/eva/themes/dark.json [hunk 16]
```diff
@@ -11,6 +11,20 @@
   "color-basic-1000": "#151A30",
   "color-basic-1100": "#101426",
 
+  "color-basic-100-transparent-8": "rgba(255, 255, 255, 0.08)",
```
Comment (artyorsh) [inline packages/eva/themes/dark.json]: I guess it makes sense to define transparent colors for **any** basic/semantic color variable and give it more generic names like `color-basic-100-level-1`, `color-basic-100-level-2`, etc

File: packages/eva/mapping.json (hunk 1 above)
Comment (32penkin) [reply packages/eva/mapping.json]: Needs discussion

File: packages/eva/mapping.json (hunk 10 above)
Comment (32penkin) [reply packages/eva/mapping.json]: Different colors - no issue.

File: packages/eva/mapping.json (hunk 11 above)
Comment (32penkin) [reply packages/eva/mapping.json]: Still issue. Needs discussion.

File: packages/eva/mapping.json (hunk 13 above)
Comment (32penkin) [reply packages/eva/mapping.json]: This isn't an issue

File: packages/eva/mapping.json (hunk 14 above)
Comment (32penkin) [reply packages/eva/mapping.json]: Different colors. Not an issue

File: packages/eva/themes/dark.json (hunk 16 above)
Comment (32penkin) [reply packages/eva/themes/dark.json]: Make sense. But on the other hand, a potential confusion is possibly related to the status (level-1, -2, etc.) of the Layout component.

**Pull request merged**